snuba/cli/jobs.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple
+
+import click
+
+from snuba.manual_jobs.ToyJob import ToyJob
+
+
+@click.command()
+@click.option(
+    "--dry_run",
+    is_flag=True,
+    default=True,
+)
+@click.argument("pairs", nargs=-1)
+def jobs(*, dry_run: bool, pairs: Tuple[str, ...]) -> None:
+    kwargs = {}
+    for pair in pairs:
+        k, v = pair.split("=")
+        kwargs[k] = v
+    job_to_run = ToyJob(dry_run, **kwargs)
```

Comment:
I think you might want to include the logic in this PR for taking in the job type as an argument and resolving it to a class. e.g. the command should be `snuba jobs run ToyJob arg1=val1`. This resolution could be done in a follow-up but I think it's the next step anyway